docs/books/modules/user-guide/message-settlement-reliability-message-routing.adoc

```diff
@@ -47,13 +47,8 @@ The following table describes the reliability guarantees for unsettled messages
 | Final disposition | Anycast | Multicast
 
 | `accepted`
-| The consumer received the message.
-a|
-Either:
-
-* All consumers received the message,
-
-* Or, at least one consumer received the message, but no consumers rejected it.
+| The consumer accepted the message.
+| At least one consumer accepted the message, but no consumers rejected it.
```

Comment:
@kgiusti This is a minor wording adjustment based on a suggestion in DISPATCH-1479. Sound OK to you?